### ISSUE TITLE
wmiobject access via "root/OpenHardwareMonitor" does not work in Windows 10

### ISSUE THREAD

**Issue by crysman** [openhardwaremonitor/openhardwaremonitor#864]
We've upgraded one machine to W10pro. Since then my powershell scripts stopped working. Theproblem is that calling

```
get-wmiobject -namespace "root/OpenHardwareMonitor" -Class Sensor
```

does not produce any output. The same script on Win server 2012R2 works just fine.

However, calling

```
get-wmiobject -namespace "root/OpenHardwareMonitor" -List
```

works.

What might be the problem? Any ideas? Thank you.

**Comment by akej74** [comment]
Probably unrelated, but I'm running the following with Python successfully on Win 10:
`hwmon = wmi.WMI(namespace="root\OpenHardwareMonitor")
sensors = hwmon.Sensor(["Name", "Parent", "Value", "Identifier"], SensorType="Temperature")`

**Comment by allstone** [comment]
Yup, WMI does not work on my PC too. Running latest cumulative updates winver 1607 build 14393. I checked and OpenHardwareMonitor does not write any sensor data under "root/OpenHardwareMonitor" at all, it is empty. Should OpenHardwareMonitor ask and receive some elevated privileges now?

**Comment by crysman** [comment]
Please, could anyone dive-in into this?

**Comment by zXLestath587Xz** [comment]
I have the same problem after update my Windows 10 x64 Build 10240 to the last Build 14393.576
(https://support.microsoft.com/en-us/help/4004227/windows-10-update-kb3206632)

Everything was working fine before update.

**Comment by crysman** [comment]
Any chance of fixing this OpenHardwareMonitor WMI API bug? :/

**Comment by dwilhelm** [comment]
I've never looked at the OpenHardwareMonitor WMI info before, so maybe I'm doing something differently, but I don't have any problem getting output on Windows 10 Pro, either with PowerShell or Python.

There's no output when OHM is not running, but that's the same as on Windows 7 for me (again with both PowerShell and Python).

In case it matters, I'm an admin on all test machines.

**Comment by DarthDestroyer** [comment]
@dwilhelm said:

1. OHM must run in the background!
2. OHM must be started as admin (but that is already necessary to receive all sensor data)

btw: the latest version is => http://openhardwaremonitor.org/files/openhardwaremonitor-v0.8.0.3-alpha.zip

**Comment by crysman** [comment]
OK, I've made further research and tests.
**Conclusion: 0.8.0 works, 0.7.1 doesn't**

I've found that we had been using an older version of OHWM (0.7.1) on one particular W10 machine, which, indeed, is not working properly. 0.7.1 does work, though, without any trouble on Windows Server 2012R2 machines.

On the other hand, the last try witch freshly downloaded 0.8.0 (yes, it is the latest stable now according to the official download page - http://openhardwaremonitor.org/downloads/) has worked now!

Thanks for your time and sorry for misleading info.
